### ISSUE TITLE
repeat this BNN model by pytorch

### ISSUE THREAD

**Issue by wxystudio** [brendanhasz/tfp-taxi#1]
Hello, I've successfully run this BNN model in tensorflow on my own data, it works very well. Now I try to repeat this model by pytorch, I find the accuracy is much lower. I'm wondering if you have ever written the pytorch version of BNN and test its performance? Thanks
